src/helpers.py

```diff
@@ -32,14 +32,22 @@ def trim_source(source: str, fn_name: str) -> str:
     return source
 
 
-def load_weights_from_file() -> Mapping[str, Any]:
+def load_weights_from_file(compiler_type: str) -> Mapping[str, float]:
```

Comment:
`load_weights_from_file` -> `get_default_randomization_weights`, to make usage of the term "randomization weights" more consistent, and because "from file" is a bit of an implementation detail

Reply:
agree that is a better name